### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by Emy-cv** [geekyutao/RN#33]
In the code, L1 loss and adversarial loss are used to train.
But in the paper, it's "We apply the same discriminators (PatchGAN (Isola et al. 2017;Zhu et al. 2017)) and loss functions (reconstruction loss, adversarial loss, perceptual loss and style loss) of the original backbone model to our model." 
Should we add perceptual loss and style loss to train your model? Or just L1 loss and adversarial loss are enough to get the numerical metric data?
Thank you for your time.

**Comment by geekyutao** [comment]
Just L1 loss and GAN loss are sufficient.
